### PR TITLE
Animals Table 🎉 🚀 ✔️ 

### PR DESCRIPTION
**Pull request by Nizamuddin4493**
# Animals Table 🎉 🚀 ✔️ 
- Attached all screenshots of all the queries below, and included all the SQL used in data.sql, queries.sql, and schema.sql.

<hr>

> Find all animals whose name ends in "mon".

<hr>

>List the name of all animals born between 2016 and 2019.

<hr>

> List the name of all animals that are neutered and have less than 3 escape attempts.

<hr>

>List the date of birth of all animals named either "Agumon" or "Pikachu".

<hr>

> List name and escape attempts of animals that weigh more than 10.5kg

<hr>

> Find all animals that are neutered.

<hr>

> Find all animals not named Gabumon.

<hr>

> Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equals precisely 10.4kg or 17.3kg)

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+INSERT INTO animals ( name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ( 'Agumon', '2-3-2020', 0, true, 10.23);
+
+INSERT INTO animals ( name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ( 'Gabumon', '9-15-2018', 5, true, 11);
```
Comment (sja-thedude) [inline data.sql]: - [ ] Good job so far. Kindly ensure that your date of birth for Gabumon should be 11-15-2018 and your weight is 8 per the requirements.

File: data.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+INSERT INTO animals ( name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ( 'Pikachu', '1-7-2021', 1, true, 15.04);
```
Comment (sja-thedude) [inline data.sql]: - [ ] Good job so far, kindly make sure that neutered value is set to false for Pikachu as per the requirements.

File: schema.sql [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        8-bit characters might not work correctly. See psql reference
+        page "Notes for Windows users" for details.
+Type "help" for help.
+
+postgres=# create database vet_clinic;
+CREATE DATABASE
+postgres=# \c vet_clinic
+You are now connected to database "vet_clinic" as user "postgres".
+
+vet_clinic=# CREATE TABLE animals (id INT GENERATED ALWAYS AS IDENTITY, name varchar, date_of_birth date, escape_attempts integer, neutered boolean, weight_kg decimal);
```
Comment (sja-thedude) [inline schema.sql]: - [ ] Good job kindly make all the data for the table displayed in different lines. Also, you don't need to write all the details as the command prompt. You could just write. For example:
```sql
 CREATE TABLE animals (
    id int,
    name varchar(255),
    date_of_birth date,
    escape_attempts integer,
    neutered boolean,
    weight_kg decimal,
   );
```

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Nizamuddin4493, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Correct Gitflow
✔️ Professional Readme

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**